conf/test.config

```diff
@@ -23,10 +23,12 @@ params {
     input = 'assets/samplesheet_test.csv'
 
     // Genome references
+    genome              = 'GRCh38'
```

Comment:
I'd recommend to test not with human as the db to download is huge.
Can you try if it annotates with C.elegans (even if variants are human).
It's what I do in Sarek